.changeset/silent-stingrays-live.md

```diff
@@ -0,0 +1,5 @@
+---
+"d1-orm": patch
```

Comment:
Thanks for the PR!

While the library is pre 1.x, I'd like (technically) breaking changes to be minor - I'd also appreciate mentioning that in the changeset 🙂

Reply:
@Skye-31 oh, ofc! updated...lmk if the message works for you.